packages/wallets/src/components/WalletListCardIcon/WalletListCardIcon.tsx

```diff
@@ -1,12 +1,51 @@
 import React from 'react';
-import './WalletListCardIcon.scss';
-
-const WalletListCardIcon: React.FC = () => {
-    return (
-        <div className='wallets-card__icon'>
-            <p> Icon (Placeholder)</p>
-        </div>
-    );
+import useDevice from '../../hooks/useDevice';
+import Bitcion from '../../public/images/bitcion.svg';
+import Demo from '../../public/images/demo.svg';
+import ETH from '../../public/images/eth.svg';
+import EUR from '../../public/images/eur.svg';
+import Tether from '../../public/images/eusdt.svg';
+import GBP from '../../public/images/gbp.svg';
+import LTC from '../../public/images/ltc.svg';
+import USD from '../../public/images/usd.svg';
+import USDC from '../../public/images/usdc.svg';
+
+const type_to_icon_mapper = {
+    USD,
+    EUR,
+    GBP,
+    BTC: Bitcion,
+    USDC,
+    ETH,
+    LTC,
+    UST: Tether,
+    eUSDT: Tether,
+    Demo,
+};
+
+const type_to_size_mapper = {
```

Comment:
@thisyahlen-deriv can we move this to the configs folder?

Reply:
I want to scope this mapper to this component. Since its only being used here.